### ISSUE TITLE
Multiple keys in HLS pack

### ISSUE THREAD

**Issue by Jason239** [google/shaka-packager#641]
Is it possible to package HLS with different keys to each tracks ? 
Is it the same way as DASH?
I know it's possible to do that by using the drm_label in DASH. 
But when it comes to HLS, how can I set the iv to each tracks ?

**Comment by kqyang** [comment]
> Is it possible to package HLS with different keys to each tracks ?

Yes. The same as DASH.

> But when it comes to HLS, how can I set the iv to each tracks ?

Unfortunately, we don't support different pre-defined IV for different tracks right now. See #292 for the tracking issue.

**Comment by kqyang** [comment]
Let us know if it answers your question. Feel free to reply if you have more questions.

**Comment by bricemiramont** [comment]
Hi,
I have to check again but what I did was package for DASH with the subtitle tracks, then package for HLS (with the same keys) but without subtitle tracks and then have a script generate a m3u playlist with 1 segment with the total length being the length of the media.
I used CBCS so iOS can use the same encrypted mp4 as shaka player.

I have some testing to do and I'll keep you informed if you want to.
